package.json

```diff
@@ -201,6 +201,11 @@
                     "type": "boolean",
                     "default": true,
                     "markdownDescription": "Enables code linting for HTML+HUBL and CSS+HUBL languages.\n\n*Requires a valid `hubspot.config.yml` file*"
+                },
+                "hubspot.neverUseHubl": {
+                    "type": "boolean",
+                    "default": false,
+                    "markdownDescription": "Disables HubL auto-detect."
```

Comment:
```suggestion
                    "markdownDescription": "Disables HubL language auto-detection globally. If this is enabled, you will never be prompted to add HTML+HubL and CSS+HubL file associations to your workspace settings. For more info see the [HubSpot VSCode documentation](https://github.com/HubSpot/hubspot-cms-vscode/tree/master#language-modes)"
```

Reply:
Also - I initially pictured this implemented with a hidden setting in `globalStorage` (same way the `workspaceStorage` works) but after thinking a bit more I like having it visible in the UI!